util/merkle-tree/Cargo.toml

```diff
@@ -1,9 +1,9 @@
 [package]
-name = "merkle-tree"
+name = "ckb-merkle-tree"
 version = "0.5.0-pre"
 license = "MIT"
 authors = ["Nervos Core Dev <dev@nervos.org>"]
 edition = "2018"
 
-[dev-dependencies]
-proptest = "0.8"
+[dependencies]
+merkle-tree = { git = "https://github.com/nervosnetwork/merkle-tree", rev="79a00ed", features = ["sha3"] }
```

Comment:
Would you publish it to crates.io and use versioned lib here?

Reply:
maybe